app/editor/src/features/content/list-view/ContentListView.tsx

```diff
@@ -117,14 +121,13 @@ export const ContentListView: React.FC = () => {
         throw error;
       }
     },
-    [api],
+    [api, filterAdvanced],
   );
 
   React.useEffect(() => {
-    fetch({ ...filter, ...filterAdvanced, sortBy });
-    // We don't want a render when the advanced filter changes.
+    fetch(filter, sortBy);
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [fetch, filter, sortBy]);
+  }, [filter, sortBy]);
```

Comment:
Don't refresh when `fetch` changes, or it'll make a call when the token is refreshed...  Not certain how to work around this.